Langs/0407.lang

```diff
@@ -69,14 +72,14 @@ Done=Fertig
 Loading WMIC=Lade WMIC
 Warning=Warnung
 [Descriptions]
-Architecture=Die Datenmenge, die Ihre CPU und Ihr Betriebssystem gleichzeitig verarbeiten können. Sollten Sie ein 32-Bit-Betriebssystem nutzen, erfordert dies eine 64-Bit Neuinstallation von Windows 11. Ihre Daten gehen dabei verloren und müssen vorab gesichert werden. Eine 32-Bit-CPU erfordert einen Wechsel auf eine 64-Bit-CPU.
-Boot=Eine Boot-Methode, die Ihr Computer zum Starten verwendet. Windows 11 benötigt den UEFI-Modus. "Legacy" bedeutet, dass sich ihr System im traditionellen Startmodus befindet. Dies kann in den BIOS/UEFI-Einstellungen korrigiert werden. Schlagen Sie im Handbuch Ihres Motherboards nach.
-CPU Name=Der Prozessor, welcher in Ihrem Computer verbaut ist. Die Liste der zu Windows 11 kompatiblen Prozessoren kann sich noch ändern. Bei einer Inkompatibilität muss der Prozessor ersetzt werden. Achtung: Dies ist bei vielen Laptops nicht möglich.
-CPU Cores=Die Anzahl der Rechenkerne, die Ihrer CPU zur Verfügung stehen. Bei einer Inkompatibilität muss die CPU ersetzt werden. Dies ist bei vielen Laptops nicht möglich.
-CPU Speed=Die Geschwindigkeit, mit der Ihre CPU die Aufgaben abarbeitet. Bei einer Inkompatibilität muss die CPU ersetzt werden. Achtung: Dies ist bei vielen Laptops nicht möglich.
+Architecture=Sollten Sie ein 32-Bit-Betriebssystem nutzen, erfordert dies eine 64-Bit-Neuinstallation von Windows 11. Ihre Daten müssen vorab gesichert werden. Eine 32-Bit-CPU erfordert einen Wechsel auf eine 64-Bit-CPU.
+Boot=Die Boot-Methode, die Ihr Computer zum Starten verwendet. Windows 11 benötigt den UEFI-Modus. Sollten Sie "Legacy" (traditioneller Startmodus) nutzen, kann dies oft in den BIOS/UEFI-Einstellungen umgestellt werden. Schlagen Sie im Handbuch Ihres Motherboards nach.
```

Comment:
I read somewhere that manufacturer provide new fw updates to enable uefi and tpm on the devices.

Maybe we can add a hint on this in general and for all languages.

_cc: @rcmaehl, @micwoj92 _